From 4940fc3a3ab5938243bf28efec3779e788bec1ed Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 08:35:50 -0700
Subject: [PATCH 05/96] HBASE-6604 Bump log4j to 1.2.17

Reason: Dependency Harmonization
Author: Jonathan Hsieh
Ref: CDH-6878
(cherry picked from commit 4102802f49d5d3285dc227a965eea07858ff6b5e)
---
 pom.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pom.xml b/pom.xml
index d637758..9fa543f 100644
--- a/pom.xml
+++ b/pom.xml
@@ -1027,7 +1027,7 @@
     <jruby.version>1.6.5</jruby.version>
     <junit.version>4.10-HBASE-1</junit.version>
     <slf4j.version>1.4.3</slf4j.version>
-    <log4j.version>1.2.16</log4j.version>
+    <log4j.version>1.2.17</log4j.version>
     <mockito-all.version>1.8.5</mockito-all.version>
     <protobuf.version>2.4.0a</protobuf.version>
     <stax-api.version>1.0.1</stax-api.version>
-- 
1.7.0.4

